src/getExtendedPublicKey.h

```diff
@@ -1,23 +1,19 @@
 #ifndef H_CARDANO_APP_GET_PUB_KEY
 #define H_CARDANO_APP_GET_PUB_KEY
 
+#include <os.h>
 #include "handlers.h"
 
 #define MAX_BIP32_PATH 10
-#define BIP_44 44
-#define ADA_COIN_TYPE 1815
-#define HARDENED_BIP32 0x80000000
 
 handler_fn_t handleGetExtendedPublicKey;
 
 typedef struct {
 	uint32_t bip32Path[MAX_BIP32_PATH];
-	uint8_t pathLength;
-	uint8_t privateKeyData[64];
+	uint32_t pathLength;
```

Comment:
I wonder whether it would make sense to create one struct
bip32Path_t

Reply:
I would not complicate it in C...

Reply:
The point is is concentrate sized arrays in one place so that we don't accidentally shoot ourselves in the foot. At least add PRIV_KEY_DATA_SIZE constant ...